### PR TITLE
WIP: Cross compile different scala versions

### PR DESCRIPTION
**Pull request by ChocPanda**

### CODE REVIEW

File: src/test/scala/org/scalacheck/magnolia/MagnoliaTest.scala [hunk 1]
```diff
@@ -94,71 +94,71 @@ object MagnoliaTest extends TestSuite {
...
-        'Set - {
+        Symbol("Set") - {
           val arbSet = Arbitrary.arbitrary[Set[Simple]]
 
           Gen.containerOf[Set, Simple](expectedSimpleGen) ==> arbSet
         }
       }
 
-      'ADTs - {
+      Symbol("ADTs") - {
```
Comment (kubukoz) [inline src/test/scala/org/scalacheck/magnolia/MagnoliaTest.scala]: Wouldn't this work with strings instead of symbols?

Comment (ChocPanda) [reply src/test/scala/org/scalacheck/magnolia/MagnoliaTest.scala]: Probably, I just went with the suggestion from the warning but I'll try it with strings

Comment (ChocPanda) [reply src/test/scala/org/scalacheck/magnolia/MagnoliaTest.scala]: Thanks @kubukoz You're right on this. Would you mind approving?

Comment (kubukoz) [reply src/test/scala/org/scalacheck/magnolia/MagnoliaTest.scala]: Sure

**Review by kubukoz — APPROVED** [review]

**Pull request merged**